### ISSUE TITLE
Version 2.0.1 breaks backwards compatibility

### ISSUE THREAD

**Issue by EECOLOR** [portabletext/react-portabletext#60]
We are useing Sanity 2.x and the webpack parser can not handle the changes introduced in this minor version.

```
Error in ./node_modules/@portabletext/react/dist/react-portable-text.esm.js
Module parse failed: Unexpected token (12:4)
You may need an appropriate loader to handle this file type.
|     marks,
|     types,
|     ...rest
|   } = overrides;
```

This most likely has to do with the rest/spread syntax. With 2.0.0 the code was this:

```js
  const {
      block,
      list,
      listItem,
      marks,
      types
    } = overrides,
    rest = _objectWithoutProperties(overrides, _excluded);
  return _objectSpread(_objectSpread({}, parent), {}, {
    block: mergeDeeply(parent, overrides, "block"),
    list: mergeDeeply(parent, overrides, "list"),
    listItem: mergeDeeply(parent, overrides, "listItem"),
    marks: mergeDeeply(parent, overrides, "marks"),
    types: mergeDeeply(parent, overrides, "types")
  }, rest);
```

With the new code it is:

```js
  const {
    block,
    list,
    listItem,
    marks,
    types,
    ...rest
  } = overrides;
  return {
    ...parent,
    block: mergeDeeply(parent, overrides, "block"),
    list: mergeDeeply(parent, overrides, "list"),
    listItem: mergeDeeply(parent, overrides, "listItem"),
    marks: mergeDeeply(parent, overrides, "marks"),
    types: mergeDeeply(parent, overrides, "types"),
    ...rest
  };
```

It seems the introduction of a major version of `@sanity/pkg-utils` has introduced this change in https://github.com/portabletext/react-portabletext/commit/32536f6b3c63aee20c1e71fc98ecb31b4936d3b7#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L52

**Comment by EECOLOR** [comment]
CC: @mariuslundgard as the commit is in your name

**Comment by EECOLOR** [comment]
This might be a workaround: https://github.com/pcbowers/sanity-plugin-tags/issues/16#issuecomment-1280134863

**Comment by snorrees** [comment]
We just released [3.0.2](https://github.com/portabletext/react-portabletext/releases/tag/v3.0.2) which should work with Sanity Studio 2
